tests/Integration/Framework/Test_ProxyDB.py

```diff
@@ -238,16 +250,16 @@ def setUpClass(cls):
     cls.userDir = tempfile.mkdtemp(dir=certsPath)
 
     # Create user certificates
-    for userName in ['no_user', 'user_1', 'user_2', 'user_3']:
+    for userName in ['no_user', 'user', 'user_1', 'user_2', 'user_3']:
       userConf = """[ req ]
-        default_bits           = 2048
+        default_bits           = 4096
         encrypt_key            = yes
         distinguished_name     = req_dn
         prompt                 = no
         req_extensions         = v3_req
         [ req_dn ]
-        C                      = DN
-        O                      = DIRAC
+        O                      = DN
+        0.O                    = DIRAC
```

Comment:
what is this `0.O` ? Why the standard `C` and `O` are not good ?

Reply:
`0.O` denotes that this is the first entry for organizationName multi-valued field.
No, `C` and `O` are good. I returned `C` to avoid confusion.